chain/yoda/execute.go

```diff
@@ -50,17 +53,37 @@ func SubmitReport(c *Context, l *Logger, id otypes.RequestID, reps []otypes.RawR
 		l.Error(":exploding_head: Failed to build tx with error: %s", err.Error())
 		return
 	}
-
-	res, err := cliCtx.BroadcastTxCommit(out)
-	if err != nil {
+	var res sdk.TxResponse
+	found := false
+	for try := 1; try <= MaxTry; try++ {
+		l.Info("Try to broadcast: %d/%d", try, MaxTry)
+		res, err = cliCtx.BroadcastTxSync(out)
+		if err == nil {
+			found = true
+			break
+		}
 		l.Error(":exploding_head: Failed to broadcast tx with error: %s", err.Error())
```

Comment:
We don't want to see this log to be critical error. There are 2 options that I think it make sense
1. Change icon to ⚠️ 
2. Change log level to debug.